### PR TITLE
CONCF-461 Basic collection view

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/CONCF-461

Collection Component uses ImageMedia components to display collection elements, as it uses lazy loading to load pics.

@hakubo @rogatty @Warkot

### CODE REVIEW

**Comment by Warkot** [comment]
LGTM

File: front/styles/main/component/_collection.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+$collection-view-image-size: 100px;
```
Comment (rogatty) [inline front/styles/main/component/_collection.scss]: Is it done only by CSS? Thumbnailer module should be used to get image URL with appropriate size. It's possible that ImageMedia does it by default but I think we have to pass some param.

File: front/scripts/main/models/ArticleModel.ts [hunk 2]
```diff
@@ -168,7 +170,8 @@ App.ArticleModel.reopenClass({
 					media: App.MediaModel.create({
 						media: article.media
 					}),
-					categories: article.categories
+					categories: article.categories,
+					collections: article.collections
```
Comment (rogatty) [inline front/scripts/main/models/ArticleModel.ts]: What if it doesn't exist?

File: front/styles/main/component/_collection.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+.collection-component {
+	text-align: center;
+
+	.article-image.visible,
+	.media-component {
```
Comment (rogatty) [inline front/styles/main/component/_collection.scss]: Isn't `.media-component` enough?

File: front/templates/main/components/collection-component.hbs [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<p> {{{header}}} </p>
```
Comment (rogatty) [inline front/templates/main/components/collection-component.hbs]: Spaces.

File: front/templates/main/components/collection-component.hbs [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+{{#each item in items}}
+	<a href={{item.link}}>
+		{{image-media url=item.img caption=item.description}}
+	</a>
```
Comment (rogatty) [inline front/templates/main/components/collection-component.hbs]: Let's keep it in one line.

File: front/styles/main/component/_collection.scss (hunk 1 above)
Comment (dianafa) [inline front/styles/main/component/_collection.scss]: it provides only height resize. I would implement also width if this code would hit the production. For now this solution seems to be ok.

**Comment by kenkouot** [comment]
May I ask what the context/use case for this is?

**Comment by dianafa** [comment]
@ken This **was** the part of our work in Modular Main Pages project. We wanted to create a new `<collection>` tag in the wikitext and then send it together with all other aticle parts i.e. media, comments, users etc. 
However, we've decided to abandon this solution and switch to Curated Content approach. So for now it won't be used.

**Pull request closed without merging**